frontend/src/Home.jsx

```diff
@@ -0,0 +1,66 @@
+import { useState, useEffect, useCallback } from "react";
+import "./Home.css";
+import SearchBar from "./components/home/SearchBar.jsx";
+import NewBoardModal from "./components/home/NewBoardModal.jsx";
+import KudosList from "./components/home/KudosList.jsx";
+
+const Home = ({ setCurrentBoardID, setCurrentBoardName }) => {
+  const [isNewBoardModalVisible, setNewBoardModalVisibility] = useState(false);
+  const [visibleBoards, setVisibleBoards] = useState([]);
+
+  const getBoards = useCallback(async () => {
+    const response = await fetch("http://localhost:3000/boards");
+    const data = await response.json();
+    setVisibleBoards(data);
+  });
+
+  useEffect(() => {
+    getBoards();
+  }, [getBoards]);
+
+  return (
+    <div className="Home">
+      <header>
+        <h1>Kudos Board</h1>
+      </header>
+
+      {isNewBoardModalVisible && (
+        <NewBoardModal
+          setNewBoardModalVisibility={setNewBoardModalVisibility}
+        />
+      )}
+      <main>
+        <div className="top-container">
+          <SearchBar />
+          <div className="select-drop">
+            <select name="seach-form" className="select-drop">
+              <option value="Sort">All</option>
+              <option value="alphabetic">Recent</option>
+              <option value="date-released">Celebration</option>
+              <option value="rating">Thank You</option>
+              <option value="rating">Inspiration </option>
```

Comment:
I'm kind of confused by these `value` values? It looks like they don't point to anything for now, so this maybe reflects the arbitrariness of what this PR does/n't cover. I'll leave that as an overall comment :)